TYPO3.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js

```diff
@@ -456,6 +458,7 @@ define(
 						that.set('editNodeTitleMode', false);
 						newNode.activate();
 						newNode.setTitle(title);
+						newNode.data.fullTitle = title;
```

Comment:
To me (as a JS noob) this looks weird. I ask myself _why doesn't setTitle() do that?_